src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.mockito.configuration;
+
+/**
+ * This is to clean runners between tests.
+ */
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
```

Comment:
ah great found it, remembered seeing this one but could remember where. You did this was groovy only? CMSClassUnloadingEnabled